experimental/uefi/baremetal-crosvm/runner

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+readonly TARGET=$1
+
+crosvm_flags=(
+  "--disable-sandbox"
+)
+
+# KVM is required for crosvm. We return 0 here to that the tests would pass.
```

Comment:
nit: second sentence seems to be missing a word, perhaps "ensure"

Reply:
Yup, thanks for noticing. Done.